### ISSUE TITLE
Amazon EKS supports cluster tagging

### ISSUE THREAD

**Issue by ewbankkit** [terraform-providers/terraform-provider-aws#10127]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

You can now add AWS tags to your Amazon EKS clusters.

### New or Affected Resource(s)

* [`aws_eks_cluster`](https://www.terraform.io/docs/providers/aws/r/eks_cluster.html)

### Potential Terraform Configuration

```hcl
resource "aws_eks_cluster" "example" {
  tags = {
    Name = "Example cluster"
  }
}
```

### References

[Announcement](https://aws.amazon.com/about-aws/whats-new/2019/09/amazon-eks-supports-cluster-tagging/).
[User guide](https://docs.aws.amazon.com/eks/latest/userguide/eks-using-tags.html).

AWS container services roadmap issue:
* https://github.com/aws/containers-roadmap/issues/82

Requires [AWS SDK v1.23.22](https://github.com/aws/aws-sdk-go/releases/tag/v1.23.22):
* https://github.com/terraform-providers/terraform-provider-aws/pull/10011

Also update:
* https://github.com/terraform-providers/terraform-provider-aws/pull/10018

**Comment by bflad** [comment]
Submitted #10307 👍

**Comment by bflad** [comment]
Support for EKS tagging has been merged and will release with version 2.31.0 of the Terraform AWS Provider, tomorrow. 🚀
